cgmes/cgmes-extensions/src/main/java/com/powsybl/cgmes/extensions/compatibility/CgmesControlAreasSerDe.java

```diff
@@ -0,0 +1,187 @@
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(CgmesControlAreasSerDe.class);
+
+    private static final String CONTROL_AREA_ROOT_ELEMENT = "controlArea";
+    private static final String CONTROL_AREA_ARRAY_ELEMENT = "controlAreas";
+    public static final String TERMINAL_ROOT_ELEMENT = "terminal";
+    public static final String TERMINAL_ARRAY_ELEMENT = "terminals";
+    public static final String BOUNDARY_ROOT_ELEMENT = "boundary";
+    public static final String BOUNDARY_ARRAY_ELEMENT = "boundaries";
+
+    public CgmesControlAreasSerDe() {
+        super("cgmesControlAreas", CgmesControlAreasSerDe.DummyExt.class, "cca",
+                new ImmutableMap.Builder<IidmVersion, ImmutableSortedSet<String>>()
+                        .put(IidmVersion.V_1_0, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_1, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_2, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_3, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_4, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_5, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_6, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_7, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_8, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_9, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_10, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_11, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_12, ImmutableSortedSet.of("1.0"))
+                        .put(IidmVersion.V_1_13, ImmutableSortedSet.of("1.0"))
+                        .build(),
+                new ImmutableMap.Builder<String, String>()
+                        .put("1.0", "http://www.powsybl.org/schema/iidm/ext/cgmes_control_areas/1_0")
+                        .build());
+    }
+
+    @Override
+    public InputStream getXsdAsStream() {
+        return getClass().getResourceAsStream("/xsd/cgmesControlAreas.xsd");
+    }
+
+    @Override
+    public List<InputStream> getXsdAsStreamList() {
+        return List.of(Objects.requireNonNull(getClass().getResourceAsStream("/xsd/cgmesControlAreas.xsd")));
+    }
+
+    @Override
+    public Map<String, String> getArrayNameToSingleNameMap() {
+        return Map.of(CONTROL_AREA_ARRAY_ELEMENT, CONTROL_AREA_ROOT_ELEMENT,
+                TERMINAL_ARRAY_ELEMENT, TERMINAL_ROOT_ELEMENT,
+                BOUNDARY_ARRAY_ELEMENT, BOUNDARY_ROOT_ELEMENT);
+    }
+
+    @Override
+    public void write(DummyExt extension, SerializerContext context) {
+        throw new UnsupportedOperationException("Backward writing compatibility is not yet supported.");
```

Comment:
`yet` means we'll do it one day, will we? Besides shouldn't this rather be something like "cannot happen"? We're here with a DummyExt which is never created, and cannot be created.